WebRoot/js/zimbraMail/calendar/view/ZmApptQuickAddDialog.js

```diff
@@ -183,34 +142,31 @@ ZmApptQuickAddDialog.prototype.isValid =
 function() {
 	var subj = AjxStringUtil.trim(this._subjectField.getValue());
 	var errorMsg = [];
-	var startDate = AjxDateUtil.simpleParseDateStr(this._startDateField.value);
-	var endDate = AjxDateUtil.simpleParseDateStr(this._endDateField.value);
+	var startDate = AjxDateUtil.simpleParseDateStr(this._appt.startDate);
+	var endDate = AjxDateUtil.simpleParseDateStr(this._appt.endDate);
 
 	if (subj && subj.length) {
-		var isValidStartEnd = DwtTimeInput.validStartEnd(this._startDateField, this._endDateField, this._startTimeSelect, this._endTimeSelect);
```

Comment:
not using DwtTimeInput.validStartEnd() anymore? Are we missing some check?

Reply:
that function was doing validation for dates also which is not needed here, so copied only the relevant part